.lintstagedrc.json

```
@@ -0,0 +1,5 @@
+{
+  "*.md": "markdownlint-cli2-fix",
+  "**/*.{ts,js,mjs,cjs}": "eslint --cache --fix",
+  "**/*.{ts,js,mjs,json,md,yml}": "prettier --cache --ignore-unknown --write"
```

```suggestion
  "*": "prettier --cache --ignore-unknown --write"
```
should fix all supported files 😉